### DISCUSSION TITLE
New FormPay which Integrate to Malaysia Payment Gateway

### DISCUSSION THREAD

**Discussion by zooster90** [buku-masjid/buku-masjid#92] [Ideas]
Integration with Toyyibpay or any other Payment gateway in malaysia

Idea:
1. Create new feature like online form pay which integrate with Toyyibpay
2. To relink this online form pay with income of the mosque with category supporting

**Comment by nafiesl** [comment]
Hello @zooster90 thanks for the idea, for this moment we are not implementing payment integration yet. But we will do in the future, because some of our users have requested that feature as well. If we implement this payment gateway integration we will try to support Indonesian payment gateways as well.

I will move this idea into discussion.

Thanks @zooster90
